SocialCareCaseViewerApi.Tests/V1/UseCase/CaseStatus/CaseStatusExecutePostUseCaseTests.cs

```diff
@@ -43,15 +44,15 @@ public void SetUp()
             _mockCaseStatusGateway
                 .Setup(x => x.CreateCaseStatus(It.IsAny<CreateCaseStatusRequest>()))
                 .Returns(caseStatus);
-
-            _mockDatabaseGateway.Setup(x => x.GetWorkerByEmail(It.IsAny<string>()))
-                .Returns(TestHelpers.CreateWorker(email: _request.CreatedBy));
         }
 
         [Test]
         public void CallsGatewaysToCheckCaseStatusExists()
         {
-            _mockDatabaseGateway.Setup(x => x.GetPersonByMosaicId(It.IsAny<long>())).Returns(TestHelpers.CreatePerson(_request.PersonId));
```

Comment:
Not necessarily for this but wondering if we should rename `GetPersonByMosaicId` to not reference Mosaic at all. What do you think? 🤔

Reply:
Agreed, I'll do this in a separate PR to not grow this PR too large.